.changes/linux-shortcuts.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": patch
+---
+
+Add global shortcut support on Linux (both x11 and wayland).
```

Comment:
```suggestion
Fix global shortcut support on Linux (both x11 and wayland).
```
We did support Linux, it was just broken and was only for x11 so I think "Fix" is better than "Add" here.